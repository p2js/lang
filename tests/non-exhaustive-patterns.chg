type Unnested = enum { A, B, C };

fn main(): i32 {
    let x = Unnested::A;
    match x {
        Unnested::A => { return 1; }
        Unnested::B => { return 2; }
    };
}
