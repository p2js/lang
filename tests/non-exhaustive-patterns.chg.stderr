
                                            throw new Error(`no check block to jump to if arm ${i} fails`);
                                            ^

Error: no check block to jump to if arm 1 fails












